packages/cloudflare-workers-bindings-extension/package.json

```diff
@@ -19,6 +19,7 @@
 		"package": "pnpm run check:type && pnpm run check:lint && node esbuild.js --production",
 		"compile-tests": "tsc -p . --outDir out",
 		"watch-tests": "tsc -p . -w --outDir out",
+		"test": "node ./out/test/runTest.js",
```

Comment:
We use `tsx` in various other places for stuff like this. Would that work? I'd alos love to see if it's possible to run this in CI—does it fail if you add a `test:ci` script?

Reply:
Just tried that but it fails with this message:

```
Error: Cannot find module '/Users/edmund/Workspace/workers-sdk/packages/cloudflare-workers-bindings-extension/src/test/suite/index'
```

I have also pushed a commit with the `test:ci` script. The tests seems to be running fine on windows and macos but fails on ubuntu (https://github.com/cloudflare/workers-sdk/actions/runs/12401086099/job/34619688337). I can look at this next as a follow-up.

Reply:
Looks like we need to have Xvfb enabled specifically for ubuntu:
https://github.com/cloudflare/workers-sdk/actions/runs/12401086099/job/34619688337